### PR TITLE
Migrate tests to buildkite

### PR DESCRIPTION
**Pull request by jaymalasinha**
Signed-off-by: Jaymala Sinha <jsinha@chef.io>

### CODE REVIEW

File: .expeditor/config.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Documentation available at https://expeditor.chef.io/
+
+# Slack channel in Chef Software slack to send notifications about build failures, etc
+slack:
+  notify_channel: delivery-cli-notify
+
+pipelines:
+  - verify
```
Comment (tduffield) [inline .expeditor/config.yml]: invalid yaml: need `- verify:`

File: VERSION [hunk 2]
```diff
@@ -0,0 +1 @@
++0.0.39
```
Comment (tduffield) [inline VERSION]: extra `+`

File: .expeditor/verify.pipeline.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
+steps:
+
+  - label: cucumber_tests_stable
+    commands:
+      - eval "$(/opt/chefdk/bin/chef shell-init bash)"
```
Comment (tduffield) [inline .expeditor/verify.pipeline.yml]: Do we need this? Did it fail without it?

**Comment by jaymalasinha** [comment]
@tduffield Thanks for catching those :)! Addressed the comments.

**Comment by afiune** [comment]
As I am trying to read this change I can't get why do we have `asdf` as a command!? 

```
- asdf global rust stable
```

I feel this is a horrible typo, maybe? @jaymalasinha @scotthain @tduffield thoughts?

Have this config/pipeline ever worked?

**Comment by scotthain** [comment]
@afiune asdf is a language installer/manager (https://github.com/asdf-vm/asdf) we use to preinstall ruby and rust versions in our default build containers/images. are you running into issues with it or just haven't seen that?

**Comment by afiune** [comment]
@scotthain Yup, we are seeing this error:
```
--
  | asdf global rust stable
  | make test in chefes/releng-base
  | bash: line 1: asdf: command not found
  | make: rustc: Command not found
  | DELIV_CLI_VERSION="0.0.39" DELIV_CLI_GIT_SHA="d578588" RUSTC_VERSION="" DELIV_CLI_TIME="2018-11-21T22:24:50Z" RUST_TEST_TASKS=1 RUST_BACKTRACE=1 GIT_COMMITTER_NAME="Chef CI" GIT_AUTHER_NAME="Chef CI" EMAIL="blackhole@chef.io" OPENSSL_INCLUDE_DIR=/usr/lib/x86_64-linux-gnu/include OPENSSL_LIB_DIR=/usr/lib/x86_64-linux-gnu cargo  test
  | /bin/sh: 1: cargo: not found
  | Makefile:87: recipe for target 'test' failed
  | make: *** [test] Error 127

```
On this PR https://github.com/chef/delivery-cli/pull/52 or any other one.

Apologies for not knowing that it is a language, I have never seen it and, honestly, it looks/sounds a bit like a typo after seeing the error message above! 😂 Right?

**Pull request merged**